documentation/glossary.md

```diff
@@ -0,0 +1,9 @@
+# Glossary
+
+:::{glossary}
+
+ForSC
+: The Fornax Science Console (ForSC) is a NASA-funded web-based application that provides access to cloud computing via JupyterLab.
+See [What is the Fornax Science Console?](intro_forsc.html).
```

Comment:
This internal link doesn't work. I see that the url slug is actually "/intro-forsc". What should the syntax here be?